add how to contribute (#8129)
PyTorchLightning/pytorch-lightning: .github/CONTRIBUTING.md

Welcome to the PyTorch Lightning community! We're building the most advanced research platform on the planet to implement the latest, best practices that the amazing PyTorch team rolls out!

> Check out our blog on how you can [start contributing](https://devblog.pytorchlightning.ai/quick-contribution-guide-86d977171b3a).

Reviewer: ```suggestion
If you are new to open source, check out [this blog to get started with your first Open Source contribution](https://devblog.pytorchlightning.ai/quick-contribution-guide-86d977171b3a).
```

## Contribution Types

> We are always looking for help to implement new features or fixing bugs.

Reviewer 2: ```suggestion
We are always open to contributors helping with new features or bug fixes.
```

> We are always open to contributors helping with new features or bug fixes.

Reviewer: ```suggestion
We are always open to contributions of new features or bug fixes.
```

3. Submit a PR!

_**Note**, even if you do not find the solution, sending a PR with a test covering the issue is a valid contribution, and we can help you or finish it with you :]_

### New Features:

1. Submit a GitHub issue - describe what is the motivation of such feature (adding the use case, or an example is helpful).

> 2. Let's discuss determining the feature scope.

Reviewer 3: ```suggestion
2. Determine the feature scope with us.
```
maybe?

   - Make sure all tests are passing.
   - Make sure you add a GitHub issue to your PR.
5. Use tags in PR name for following cases:

>    - **[blocked by #<number>]** if your work is depending on others changes.

Reviewer 3: ```suggestion
   - **[blocked by #<number>]** if your work is dependent on other PRs.
```
